schemas/v3.0/schema.yaml

```diff
@@ -331,7 +331,12 @@ definitions:
     patternProperties:
       '^x-': {}
     additionalProperties: false
-
+    dependencies:
+      discriminator:
+        anyOf:
+          - required: [allOf]
+          - required: [anyOf]
+          - required: [oneOf]
```

Comment:
I'm not sure you meant for this to go in this PR, it seems to not fit with the title/description.

If you did: I don't know `discriminator` well, but the example [Models with Polymorphism Support](https://github.com/OAI/OpenAPI-Specification/blob/3.0.3/versions/3.0.3.md#models-with-polymorphism-support) has `discriminator` in `/components/schemas/Pet` without a `*Of` keyword. But then `/components/schemas/Pet` is never used in the example in a way that would invoke `discriminator`, so that's confusing.

Reply:
@notEthan oops- looks like I need to rebase this (there was a whole thing where this got added and then removed), doing so momentarily.

Reply:
It's rebased but seems to be the same as what I commented on

Reply:
wat... oh %$@#! I fixed it locally and forgot to add that commit.  I'm still not sure how it got there in the first place.  Fixed for real now, sorry about that 🤦